src/main/java/com/werfen/report/service/pdf/util/PdfExportService.java

```diff
@@ -23,6 +24,19 @@ public File export(JasperPrint jasperPrint, String filePath) throws JRException
         return new File(filePath);
     }
 
+    public ByteArrayOutputStream exportToStream(JasperPrint jasperPrint) throws JRException {
```

Comment:
Considering that the already `export` method is not used from outside this library, I would suggest renaming it to `exportToFile`, so no export method has greater importance than the other.

Reply:
Agree, it will be renamed in the following commit.